src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,354 @@
+                      setRegionsList(['allRegions']);
+                    }
+                  }}
+                >
+                  [
+                  <MenuItem
+                    style={{ color: 'black' }}
+                    key="allRegions"
+                    value="allRegions"
+                  >
+                    All
+                  </MenuItem>
+                  <ListSubheader>Administrative Regions</ListSubheader>
+                  ...
+                  {regionNames.map(region => {
+                    return (
+                      <MenuItem
+                        style={{ color: 'black' }}
+                        key={region}
+                        value={region}
+                      >
+                        {region}
+                      </MenuItem>
+                    );
+                  })}
+                  ]
+                </Select>
+              </div>
+              <div className={classes.AlertFormOptions}>
+                <TextField
+                  id="email-address"
+                  className="emailAddressClass"
+                  label="Email Address"
+                  type="text"
+                  variant="filled"
+                  onChange={onChangeEmail}
+                />
+              </div>
+            </div>
+            <Button
+              className={classes.innerAnalysisButton}
+              onClick={runAlertForm}
+              disabled={!hazardLayerId || !selectedStat || !emailValid}
+            >
+              <Typography variant="body2">Create Alert</Typography>
+            </Button>
+          </div>
+        ) : null}
+      </Box>
+    </div>
+  ) : null;
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    AlertForm: {
+      zIndex: theme.zIndex.drawer - 1, // position this below the Analysis drawer
+      position: 'absolute',
+      top: 40,
+      left: 2,
+      textAlign: 'left',
+    },
+    AlertFormMenu: {
+      backgroundColor: '#5A686C',
+      maxWidth: '100vw',
+      color: 'white',
+      overflowX: 'hidden',
+      whiteSpace: 'nowrap',
+      borderTopRightRadius: '10px',
+      borderBottomRightRadius: '10px',
+      height: 'auto',
+      maxHeight: '60vh',
+    },
+    AlertFormButton: {
+      height: '36px',
+      'margin-left': '3px',
+    },
+    AlertFormOptions: {
+      padding: '5px 0px',
+    },
+    newAlertFormContainer: {
+      padding: '5px',
+      marginTop: '10px',
+    },
+    radioOptions: {
+      '&.Mui-checked': { color: grey[50] },
+      padding: '2px 10px 2px 20px',
+    },
+    innerAnalysisButton: {
+      backgroundColor: '#3d474a',
+      margin: '10px',
+      '&.Mui-disabled': { opacity: 0.5 },
+    },
+    selectorLabel: {
+      '&.Mui-focused': { color: 'white' },
+    },
+    selector: {
+      margin: '5px',
+    },
+    numberField: {
+      paddingLeft: '10px',
+      marginTop: '10px',
+      width: '85.5px',
+      '& .Mui-focused': { color: 'white' },
```

Comment:
We should avoid using `& ...Mui class-name` since its a little prone to breaking. Overriding this style in the global theme (where we have access to TS checking) is the best route. If you don't want this style to apply globally, then we could keep this. 
Also applies to the 2 other cases in this file.

Reply:
I was able to delete 2 of the 3 occurrences here. I think putting this in the global theme is a good idea.